api/v1/requests/AttendeeRequest.js

```diff
@@ -45,10 +41,10 @@ const bodyValidations = {
   'attendee.isPrivate': attendee.validations.isPrivate,
   'attendee.hasLightningInterest': [ 'boolean' ],
   'attendee.phoneNumber': attendee.validations.phoneNumber,
-  'ecosystemInterests': ['array', 'maxLength:4', validators.array(validators.nested(ecosystemInterestValidations, 'ecosystemInterests'))],
-  'projects': ['array', 'maxLength:1', validators.upTo(['isSuggestion', false], 1, projectErrorMessage), validators.array(validators.nested(projectValidations, 'projects'))],
-  'extras': ['array', 'maxLength:1', validators.array(validators.nested(extraInfoValidations, 'extras'), 'extras')],
-  'collaborators': ['array', 'maxLength:8', validators.array(validators.nested(requestedCollaboratorValidations, 'collaborators'))]
+  'longForm': ['array', 'maxLength:1', validators.array(validators.nested(longFormValidations, 'longForm'), 'longForm')],
+  'collaborators': ['array', 'maxLength:8', validators.array(validators.nested(requestedCollaboratorValidations, 'collaborators'))],
+  'extraInfo': ['array', 'maxLength:2', validators.array(validators.nested(extraInfoValidations, 'extraInfo'), 'extraInfo')],
```

Comment:
Why are we allowing them to add 2 extrainfos?

Reply:
Registration doc said websites should be limited to a max of 2.